src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -702,15 +702,14 @@ private void flush(boolean commitTranslog, boolean force, boolean waitIfOngoing)
                 if (commitTranslog) {
```

Comment:
Pre-existing, and we should fix this later, but: do we really need this commitTranslog boolean anymore?  Can't we just always commit the translog, now that recovery is able to pull its own snapshot and is decoupled from flushing?